index.html

```diff
@@ -40,9 +48,13 @@ <h3 class="presentation">I'm a Software Developer</h3>
     </ul>
   </section>
   <section class="works">
-    <h1 class="works_title">My Recent Works</h2>
-      <div class="project1">
-        <img src="./images/Yoga.svg" alt="yoga pose" class="arrow"></img>
+    <div class="works_div">
```

Comment:
- [ ] As shown in the images below using the red lines you have inconsistent styling with the provided figma template for this section please kindly work on designing your portfolio webpage better so it closely match the provided figma template as shown below :+1: :sparkles: 
- figma design
![1-w](https://github.com/ommarz/Personal_Portfolio/assets/89055502/fce96f1b-2d4c-40a1-9e3b-25eb2583c9bd)
- your design
![temp-2-w-1](https://github.com/ommarz/Personal_Portfolio/assets/89055502/67648e86-92f6-460d-9ef6-1386f6e79d7b)